Core2/gradle/core2.versions.toml

```diff
@@ -1,7 +1,16 @@
 [versions]
-sentry-android = "7.15.0"
+coreKtx = "1.15.0"
+composeBom = "2024.11.00"
 matomo = "4.1.4"
+sentry-android = "7.15.0"
 
 [libraries]
-sentry-android = { module = "io.sentry:sentry-android", version.ref = "sentry-android" }
+androidx-core-ktx = { group = "androidx.core", name = "core-ktx", version.ref = "coreKtx" }
+compose-foundation = { group = "androidx.compose.foundation", name = "foundation-android" }
+compose-material3 = { group = "androidx.compose.material3", name = "material3-android" }
+compose-runtime = { group = "androidx.compose.runtime", name = "runtime-android" }
+compose-ui-tooling = { group = "androidx.compose.ui", name = "ui-tooling" }
+compose-ui-tooling-preview = { group = "androidx.compose.ui", name = "ui-tooling-preview-android" }
+compose-bom = { group = "androidx.compose", name = "compose-bom", version.ref = "composeBom" }
```

Comment:
Almost all of them are already available in `libs`, no need to add them in `core2`.

Reply:
We want them in Core2 to have the Core2 module be independent from the SwissTransfer module.

An alternative though could be to define SwissTransfer's dependencies based on the Core2 ones. Maybe this needs to be thought through just to be sure but this would recreate the same ecosystem we had with the original Core as well